### ISSUE TITLE
The screen size and time zone spoofing doesn't work anymore.

### ISSUE THREAD

**Issue by TheHighborne** [dillbyrne/random-agent-spoofer#150]
I'm fairly new to this add-on, so I'm not sure if it's just an error on my side, but Firefox 36 seems to have caused quite a few bugs. I installed RAS today after the update and the ability to spoof the time zone, the screen size and possibly other options are not working correctly. I tried different settings for both options, but the test scripts on browserspy.dk give me always the same results. I'm running a 64-bit Windows 7 system and I have NoScript and ublock installed, I also made some unrelated changes to about:config.

The userAgent switcher works as intended and I can't assess the state of the more advanced options.

The reason why I blame Firefox 36 is that the add-on worked in the past on my system and the update is the most obvious reason for the sudden change. The last time I had it installed was roughly one month ago.

**Comment by dillbyrne** [comment]
Hello @TheHighborne . It is still working fine for me with firefox 36. The only issue I have noticed was with some notifications not showing up but windows is not affected by that and I have only had a chance to test linux and I think that could be a problem with the sdk library itself and what ever way it interfaces with libnotify.

The screen spoofing and some other options use script injection to work and they get injected as the first script element in each page so if there is other addons which also do similiar the might cause problems.

I'm using it along side NoScript and it works fine. if scripts are allowed on a page by noscript then the RAS injection script should run. 

I'd suggest to try it in a [fresh profile](https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles) and it should work. Then add your addons one by one and see what is causing it.

I assume you are using the github version as the script injection in not present in the mozilla version

**Comment by TheHighborne** [comment]
This is strange, I added a new profile and everything worked as intended. Then I started removing some add-ons and scripts from my default profile and suddenly the options worked as well. I readded every single thing I removed but the problem is still gone. I couldn't locate the source, even after trying the whole process a second time, perhaps it was related to the install order of some scripts, but I don't know.
The only problem that remains is that disabling the local DOM storage breaks some scripts, but that's probably unavoidable. 
So, thanks for your help and for this great add-on and have a great weekend!
